docs/API.md

```diff
@@ -1702,10 +1703,29 @@ The above example is equivalent to the `meta/common-info` example above.
 
 **Not yet implemented**
 
-#### POST *id*/resources/name.stream
+#### GET *id*/meta/resources
```

Comment:
Yes, it was deemed unnecessary to mention meta/any in all meta endpoints.
Every meta endpoint automatically gets the benefit of bulk call support.